apps/studio/components/layouts/ProjectLayout/LayoutHeader/LocalVersionPopover.tsx

```diff
@@ -0,0 +1,168 @@
+          {isLatestVersion ? 'Latest' : 'Update available'}
+        </Badge>
+      </PopoverTrigger_Shadcn_>
+      <PopoverContent_Shadcn_ align="end" className="w-80 px-0">
+        {hasLatestVersionCheck ? (
+          !isLatestVersion ? (
+            <div className="px-4 mb-3">
+              <p className="text-sm mb-2">
+                A new version of Supabase Studio is available and can be updated via the CLI:
+              </p>
+              <Tabs_Shadcn_ defaultValue="macos">
+                <TabsList_Shadcn_ className="mt-2">
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="macos">
+                    macOS
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="windows">
+                    Windows
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="linux">
+                    Linux
+                  </TabsTrigger_Shadcn_>
+                  <TabsTrigger_Shadcn_ className="px-2 text-xs" value="npm">
+                    npm / Bun
+                  </TabsTrigger_Shadcn_>
+                </TabsList_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="macos">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="windows">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    scoop upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="linux">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="npm">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    npm update supabase --save-dev
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+              </Tabs_Shadcn_>
+            </div>
+          ) : (
+            <div className="px-4 mb-3">
+              <p className="text-sm">You're on the latest version of Studio</p>
+            </div>
+          )
+        ) : null}
+
+        <div className="flex flex-col gap-y-2 px-4">
+          <p className="text-xs text-foreground-lighter">
+            All available image versions of Supabase Studio can be found on our{' '}
+            <InlineLink href="https://hub.docker.com/r/supabase/studio/tags">Docker Hub</InlineLink>
+            .
+          </p>
+        </div>
+
+        <div className="flex items-center gap-x-2 mt-3 px-4">
+          <Dialog>
```

Comment:
There might be a good ui-pattern component for this. Think we have a "Modal" ui pattern component.

Reply:
hmm we do have a Modal component but its labelled as deprecated and to use Dialog instead?
<img width="352" alt="image" src="https://github.com/user-attachments/assets/120c9195-cca1-4cf7-b4de-789702c898d3" />